### PR TITLE
[15.0][MIG] sales_team_security_crm: Migration to 15.0

### PR DESCRIPTION
**Pull request by cesar-tecnativa**
@Tecnativa TT36547

### CODE REVIEW

**Comment by pedrobaeza** [comment]
You can rebase now

**Review by pedrobaeza — COMMENTED** [review]
Same comments as the other PR

**Comment by cesar-tecnativa** [comment]
@chienandalu could you review please?

**Comment by pedrobaeza** [comment]
Tests are red

File: sales_team_security_crm/security/security.xml [hunk 1]
```diff
@@ -1,40 +1,35 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<odoo>
-    <data noupdate="1">
```
Comment (pedrobaeza) [inline sales_team_security_crm/security/security.xml]: I have run in local pre-commit, and this is not being modified by it, so better to keep the same 2 level indentation for not having too much noise in the diff.

**Review by pedrobaeza — APPROVED** [review]

**Comment by cesar-tecnativa** [comment]
@CarlosRoca13 please review

**Comment by cesar-tecnativa** [comment]
@LudLaf Please review

**Comment by LudLaf** [comment]
Readme file can be enriched, so main module's functionality is depicted in a clear way.

**Review by CarlosRoca13 — APPROVED** [review]

**Pull request merged**